src/chat/components/CustomImageVideoBubble.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useRef, useState } from 'react';
+import {
+  StyleSheet,
+  View,
+  Pressable,
+  Image,
+  TouchableOpacity,
+} from 'react-native';
+import Video, { VideoRef } from 'react-native-video';
+import FastImage from 'react-native-fast-image';
+import { MessageTypes, type MessageProps } from '../../interfaces';
+
+interface CustomImageVideoBubbleProps {
+  message: MessageProps;
+  position: 'left' | 'right';
+  selectedImgVideoUrl: (url: string) => void;
```

Comment:
I think this prop name should begin with a verb, so it better be `onPress` or `onSelectCallback`